modules/enet/SCsub

```diff
@@ -25,7 +25,6 @@ if env["builtin_enet"]:
 
     env_enet.Prepend(CPPPATH=[thirdparty_dir])
     env_enet.Append(CPPDEFINES=["GODOT_ENET"])
-    env_enet.Append(CPPDEFINES=["ENET_DEBUG"])
```

Comment:
I would prefer to comment this line, so in case we have to use it again we don't need to search for this macro name.

Reply:
Got it. I will keep the name in a comment, but I try to keep a policy of leaving code out of comments, because it can quickly grow out of sync and is best seen in the context of points in the file's history.